server/singer/parser.go

```diff
@@ -84,7 +87,7 @@ func StreamParseOutput(stdout io.ReadCloser, consumer PortionConsumer, logger lo
 			}
 
 			outputPortion.State = state
-
+			outputPortion.NeedClean = !initialized && strings.Contains(string(lineBytes), "FULL_TABLE")
```

Comment:
It seems that not state, but [catalog](https://github.com/singer-io/getting-started/blob/master/docs/SPEC.md#catalog) contains replication method (incremental or full table)

Reply:
fixed